### PR TITLE
user association has been created with tests

### PR DESCRIPTION
**Pull request by kennyfrc**

### CODE REVIEW

File: spec/card_spec.rb [hunk 1]
```diff
@@ -44,5 +45,11 @@
         expect(card.right_translation?("Hello")).to eq(false)
       end
     end
+
+    describe "user" do
+      it "validates that the card belongs to a user" do
+        expect(card.user.name).to eq("kenn")
+      end
+    end
```
Comment (davetoxa) [inline spec/card_spec.rb]: @kennyfrc useless test :) you don't need to test user here :)

File: spec/factories/card.rb [hunk 2]
```diff
@@ -3,5 +3,6 @@
     original_text "How are you?"
     translated_text "Kamusta?"
     review_date DateTime.now.getutc
+    user_id 1
```
Comment (davetoxa) [inline spec/factories/card.rb]: @kennyfrc `user`

File: spec/user_spec.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+describe "User" do
+  let(:user) { create(:user) }
+
+  describe "#cards" do
+    it "user has many cards" do
+      expect(user.cards).not_to eq(0)
+    end
+  end
+end
```
Comment (davetoxa) [inline spec/user_spec.rb]: @kennyfrc useless test too :)

File: spec/card_spec.rb (hunk 1 above)
Comment (kennyfrc) [reply spec/card_spec.rb]: @davetoxa do you have a guideline of what methods to test? kind of struggling to decide what i should test vs what i shouldn't test

Comment (kennyfrc) [reply spec/card_spec.rb]: maybe i should only test methods that are in the Model? or if it's related to user input?

File: spec/factories/card.rb (hunk 2 above)
Comment (kennyfrc) [reply spec/factories/card.rb]: @davetoxa wow.. i'm surprised it works. how did that work? the others are obvious (method call + an argument) but i'm confused how passing an object is magically understood by Rails.

what does each line of code within the block return here?

File: spec/card_spec.rb (hunk 1 above)
Comment (davetoxa) [reply spec/card_spec.rb]: only model/class methods you should test )

File: spec/factories/card.rb (hunk 2 above)
Comment (davetoxa) [reply spec/factories/card.rb]: magic =)

**Pull request merged**